### DISCUSSION TITLE
Multi language support

### DISCUSSION THREAD

**Discussion by Anil-Kumar-k** [atrocore/atrocore#527] [Q&A]
Hello,

Is it possible to enable different indian language supportin atrocore pim?If yes could you please guide me to enable. Do we have to install any packages etc. Please let me know.

Thanks in advance.

Regards
Anil

**Comment by atrolex** [comment]
Hi Anil, we need to add it to the core. Which one do you need?

**Reply by Anil-Kumar-k** [reply]
Thank you very much for the response. looks like we are able to resolve the issue by adding Hindi language from Admin console. In case of any other issue, I will reach out to you. However if you have any documentation regarding language, kindly share the link.
